docs/how-to/package-with-hatch.md

```diff
@@ -0,0 +1,385 @@
+# Package a Textual app with Hatch
+
+Python apps may be distributed via [PyPI](https://pypi.org/) so they can be installed via `pip`.
+This is known as *packaging*.
+The packaging process for Textual apps is much the same as any Python library, with the additional requirement that we can launch our app from the command line.
+
+!!! tip
+
+    An alternative to packaging your app is to turn it into a web application with [textual-web](https://github.com/Textualize/textual-web).
+
+In this How To we will cover how to use [Hatch](https://github.com/pypa/hatch) to package an example application.
+
+Hatch is a *build tool* (a command line app to assist with packaging).
+There are other build tools, such as [Poetry](https://python-poetry.org/).
+Packaging with Poetry is similar to Hatch, so you may want to read this even if you are a Poetry user.
```

Comment:
This post isn't far off from being a hatch tutorial - if someone is a beginner and is using Poetry, I feel like reading this post will be confusing, and telling them that they may want to read it might set up expectations that it's going to be more helpful than it is.